examples/helia-101/102-cid-profiles.js

```diff
@@ -0,0 +1,52 @@
+import { createReadStream } from 'node:fs'
+import { unixfs } from '@helia/unixfs'
+import { createHelia } from 'helia'
+import { fixedSize } from 'ipfs-unixfs-importer/chunker'
+import { balanced } from 'ipfs-unixfs-importer/layout'
+
+
+// set up
+
+const helia = await createHelia()
+const fs = unixfs(helia)
+
+// using a local copy of https://www.gutenberg.org/files/2600/2600-h/2600-h.htm , curled to this directory but not checked in
+const bigHtmlFile = createReadStream('./2600-h.htm')
+
+// generate CID according to Helia defaults (~== kubo v1 profile) from bigHtmlFile:
+const cidBigHTML = await fs.addFile(
```

Comment:
You can simplify this by having helia download the file for you as I did in https://github.com/ipfs-examples/helia-examples/pull/456/files#diff-15db2a7c59c6093dbf74da5c6c0132ad792799c35a93eabe589cf6cd00d1019dR65

Reply:
i thought having a local file would simplify the comparison to kubo, since i don't think the latter has a "from URL" option?  this is neat, tho, maybe it makes sense to add it as a 5th example before the local-file version to show that it's the same CID between `urlSource(url)` and the local file curled from the same URL... i leave it to the tutorials' editor to decide!